docs/articles/documentation/extending-testcafe/custom-browser-provider-plugin/browser-provider-methods.md

```diff
@@ -0,0 +1,130 @@
+---
+layout: docs
+title: Browser Provider Methods
+permalink: /documentation/extending-testcafe/custom-browser-provider-plugin/browser-provider-methods.html
+checked: false
+---
+# Browser Provider Methods
+
+To create your [custom browser provider](index.md#implementing-the-browser-provider), implement the following provider methods.
+
+## Required Methods
+
+### openBrowser
+
+Runs the specified browser and opens the specified page in it.
+
+```text
+openBrowser(id, browserName, startPage)
+```
+
+Parameter     | Type   | Description
+------------- | ------ | ----------------------------
+`id`          | String | The browser identifier.
+`browserName` | String | The browser name.
+`startPage`   | String | The url of the page to open.
+
+See [Example](index.md#example).
+
+### closeBrowser
+
+Closes the specified browser.
+
+```text
+closeBrowser(id, pageInfo)
+```
+
+Parameter   | Type   | Description
+----------- | ------ | -----------------------------------
+`id`        | String | The browser identifier.
+`pageInfo`  | Object | The [pageInfo](#pageinfo) object.
```

Comment:
We need to describe it as well. Because from use perspective it's not obvious for me why I need `title` when I have e.g. `id`. BTW, don't we set `title` to `id` on actions?